dev-scheme/chicken/chicken-4.13.0.recipe

```diff
@@ -0,0 +1,120 @@
+SUMMARY="A Scheme interpreter and native Scheme to C compiler"
+DESCRIPTION="CHICKEN is a compiler for the Scheme programming language. \
+CHICKEN produces portable and efficient C, supports almost all of the R5RS \
+Scheme language standard, and includes many enhancements and extensions. \
+CHICKEN runs on Linux, MacOS X, Windows, and many Unix flavours."
+HOMEPAGE="http://www.call-cc.org/"
+REVISION="1"
+LICENSE="BSD (2-clause)"
+COPYRIGHT="2008-2017 The Chicken Team
+		2000-2007 Felix L. Winkelmann"
```

Comment:
```diff
-REVISION="1"
 COPYRIGHT="2008-2017 The Chicken Team
-		2000-2007 Felix L. Winkelmann"
+	2000-2007 Felix L. Winkelmann"
+LICENSE="BSD (2-clause)"
+REVISION="1"
```
(recommended ordering, according to [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering))